src/core/Table/Table.test.tsx

```diff
@@ -2844,3 +2844,32 @@ it('should show column enabled when whole row is disabled', () => {
   expect(rowCells[0].classList).not.toContain('iui-disabled');
   expect(rowCells[1].classList).toContain('iui-disabled');
 });
+
+it('should render selectable rows without select column', () => {
+  const onRowClick = jest.fn();
+  const { container, getByText } = renderComponent({
+    isSelectable: true,
+    selectionMode: 'single',
+    onRowClick,
+  });
+
+  expect(screen.queryByText('Header name')).toBeFalsy();
```

Comment:
Why this is needed?

Reply:
Found it in the `should handle row clicks` test, but not sure why it's needed. Removed